.travis.yml

```diff
@@ -2,6 +2,7 @@
 dist: trusty
 addons:
   chrome: stable
+  firefox: latest-beta
```

Comment:
Can you replace `latest-beta` with `latest`? I don't want to deal with betas.

Reply:
`Beta` is very close to `latest`. It make sense not to use `dev` or `nightly` because they are instable, but it's good to test against upcoming releases and fix issues before users will face them in production environment.

Reply:
If `beta` was as stable as `latest` it wouldn't be a `beta`. I don't want to deal with betas.

Reply:
> I don't want to deal with betas.

It's not about you or me, it's about catching issues before real users will experience them in production apps.

I will repeat myself: end-users experience issues in apps with `cash` as a dependency. That's the least we want, right?

If so, it might make sense to experience some rare instabilities for two of us, rather than let (thousands/millions/?) users experience issues.

Reply:
> It's not about you or me

It is (also) about me since if the build fails I'll have to deal with it.

> I will repeat myself: end-users experience issues in apps with cash as a dependency. That's the least we want, right?

What's your point here, are you saying that they experience issues and don't report them? Or that they experience issues and nobody knows about it because they are running a beta?

> If so, it might make sense to experience some rare instabilities for two of us, rather than let (thousands/millions/?) users experience issues.

It's not like the `classList` API or whatever will be deprecated tomorrow or the browsers want to break the web. Overall I think it's way more likely that I'll publish a regression by mistake.

[Major fuck-ups](https://stackoverflow.com/questions/52390368/array-state-will-be-cached-in-ios-12-safari-is-it-a-bug-or-feature/52392901#52392901) still happen, and I've no interest whatsoever in working around those.

I'll tell you what, if you can find an instance, that happened in the last year, where a major browser pushed a breaking change to it's stable channel, and kept it in subsequent releases I'll accept testing on beta versions.

Reply:
> I'll tell you what, if you can find an instance, that happened in the last year, where a major browser pushed a breaking change to it's stable channel, and kept it in subsequent releases I'll accept testing on beta versions.

I can't come up with any right now, `:%s/latest-beta/latest/` :)